dist/src/main/config/broker.yaml.template

```diff
@@ -201,6 +201,100 @@
+      # enabled : true
+
+      # if enabled - will use the average latencies over a window as the current latency to update the limit.
+      # It is not recommended to enable this when the algorithm is aimd. This setting is not applicable to fixed limit.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_USEWINDOWED
+      # useWindowed: true
+
+      # The algorithm configures which algorithm to use. It should be one of vegas, aimd, fixedLimit, gradient, or gradient2.
+      # You then only need to configure the subsection relating to the algorithm of your choice.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_ALGORITHM
+      # algorithm: "vegas"
+
+      # Configure the following if the algorithm is "aimd".
+      # AIMD increases the limit for every successfully response and decrease the limit for every request timeout.
+      # aimd:
+        # The limit will be reduced if the observed latency is greater than the requestTimeout.
+        # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_AIMD_REQUESTTIMEOUT
+        # requestTimeout: "1s"
+
+        # The initial limit to be used when the broker starts. The limit will be reset to this value when the broker restarts.
+        # This setting can also be overridden using the environment ZEEBE_BROKER_BACKPRESSURE_AIMD_INITIALLIMIT
+        # initialLimit: 100
+
+        # The minimum limit. This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_AIMD_MINLIMIT
+        # minLimit: 1
+
+        # The maximum limit. This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_AIMD_MAXLIMIT
+        # maxLimit: 1000
+
+        # The backoffRatio is a double value x such that 0 <  x  < 1. It determines the factor by which the limit is decreased.
+        # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_AIMD_BACKOFFRATIO
+        # backoffRatio: 0.9
+
+      # Configure the following if the algorithm is "fixed"
+      # fixedLimit:
+        # Set a fixed limit. This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_FIXEDLIMIT_LIMIT
+        # limit: 20
+
+      # Configure the following if the algorithm in "vegas"
+      # Vegas is an adaptive limit algorithm based on TCP Vegas congestion control algorithm.
+      # It estimates a queue size which indicates how many additional requests are in the queue over the estimated limit.
+      # The limit is adjusted based on this queueSize.
+      # vegas:
+        # The initial limit to be used when the broker starts. The limit will be reset to this value when the broker restarts.
+        # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_VEGAS_INITIALLIMIT
+        # initialLimit: 20
```

Comment:
a bit confused here: 
- the initial limit is 20
- beta is 6 and according to some of the next lines it means the limit will be decreased

So why start with a limit of 20 if we immediately throttle it down to 6?

Maybe I am misunderstanding something.

Reply:
"limit" and "queuesize" are different things. limit is the maximum number of inflight requests. queuesize is something specific to vegas algorithm. The limit is adjusted based on the queuesize.